src/commonMain/kotlin/exchange.dydx.abacus/state/app/helper/TriggerOrderToastGenerator.kt

```diff
@@ -348,17 +371,21 @@ class TriggerOrderToastGenerator(
         }
 
         val orderType = orderTypePath?.let { localizer.localize(it) }
-        val detail = detailStringPath?.let {
+        var detail = detailStringPath?.let {
             localizer.localizeWithParams(
                 path = it,
                 params = mapOf("NEW_VALUE" to triggerPrice),
             )
         }
+        val failedReasons = status.failedReason
+        if (failedReasons != null) {
+            detail += " $failedReasons"
```

Comment:
Does this coming the localized error message with the broadcast error?

Reply:
Yup.  Web does its own message and this code path is for mobile only